src/services/sequelize/repositories/transaction.ts

```diff
@@ -7,10 +7,13 @@ import { Logger } from "@server/loaders"
 
 import {
     MetadataAttributes,
+    Order,
+    Quote,
     Transaction as TransactionModel,
     TransactionAttributes,
     TransactionMetadata
 } from "../models"
+const { Op } = require("sequelize")
```

Comment:
why are we using require and not import statement here?
```suggestion
import { Op } from "sequelize"
```

Reply:
Fixed this already, was a silly mistake